test/test_helper.rb

```diff
@@ -1,18 +1,31 @@
+# Setup RAILS_ENV as test and expand config for test environment
 ENV["RAILS_ENV"] = "test"
 require File.expand_path("../../config/environment", __FILE__)
-require "rails/test_help"
-require "minitest/rails"
-require 'database_cleaner'
-require "minitest/pride"
-require "minitest/autorun"
-require "minitest/osx"
 
-# Require help with all tests
-require 'helpers/auth_helper'
-require 'helpers/assert_helper'
+# Check if we're connected to the test DB
+begin
+  ActiveRecord::Base.connection
+rescue ActiveRecord::NoDatabaseError
+  # No database... try setting up
+  puts "No test database has been setup! Setting first-time up..."
+  system "RAILS_ENV=test rake db:setup db:init_test_data"
```

Comment:
Likewise. It was originally there to begin with but I think Jake and I had some issues trying to get that working a while back. Let me give it another go :)

Sent from my iPhone

> On 8 Jul 2016, at 9:11 PM, Andrew Cain <notifications@github.com> wrote:
> 
> In test/test_helper.rb:
> 
> >  
> > -# Require help with all tests
> > -require 'helpers/auth_helper'
> > -require 'helpers/assert_helper'
> > +# Check if we're connected to the test DB
> > +begin
> > +  ActiveRecord::Base.connection
> > +rescue ActiveRecord::NoDatabaseError
> > +  # No database... try setting up
> > +  puts "No test database has been setup! Setting first-time up..."
> > +  system "RAILS_ENV=test rake db:setup db:init_test_data"
> @alexcu Have a quick check -- but I recall finding a way of running rake tasks from rails without needing a system call... if we can change this that would be better.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>